hiworkload/compile.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
 # SPDX-License-Identifier: GPL-2.0-or-later
-# Copyright (c) 2019 Petr Vorel <pvorel@suse.cz>
+# Copyright (c) 2019-2022 Petr Vorel <pvorel@suse.cz>
 set -e
 
+missing=
+for cmd in aclocal autoconf autoheader automake make; do
+	if ! command -v $cmd >/dev/null; then
+		echo "Missing '$cmd', install it!"
+		missing=1
+	fi
+done
+
+if [ "$missing" ]; then
+	exit 99
```

Comment:
There is exit code 22 used for skipped elsewhere in the code. Do we want to stick to that convention?